etc/openwrt/openthread-br/README.md

```diff
@@ -0,0 +1,71 @@
+OpenThread Border Router on OpenWRT
+===================================
+
+## Build
+
+This is for local development.
+
+### 1. Add OpenThread feed
+
+Assuming `OPENWRT_TOP_SRCDIR` is root of openwrt sources.
+
+```bash
+echo src-link openthread "$(pwd)/etc/openwrt" >> ${OPENWRT_TOP_SRCDIR}/feeds.conf
+cd "${OPENWRT_TOP_SRCDIR}"
+./scripts/feeds update openthread
+./scripts/feeds install openthread-br
+```
+### 2. Enable OpenThread Border Router
+
+OpenThread is not selcted by default, so use menuconfig to select openthread-br(OpenThread Border Router).
+
+```bash
+make menuconfig
+```
+
+In the configure window, use key Up, key Down to move cursor and key Left, key Right choose action.
```

Comment:
...use the Up and Down keys to move the cursor, and the Left and Right keys to choose an action.